install.sh

```diff
@@ -2,9 +2,7 @@
 
 set -e
 
-# FIXME: change to latest version after lightning fix issue
-# https://github.com/pingcap/tidb-lightning/issues/277
-TAG="v3.1.0-beta.1"
+TAG="nightly"
```

Comment:
seems can't use `"nightly"` here. what about `"latest"`?

Reply:
TiDB branch doesn't have a "nightly" tag. I add a single TiDB_TAG for "v4.0.4" now